containers/sarek/environment.yml

```diff
@@ -0,0 +1,24 @@
+# You can use this file to create a conda environment for this pipeline:
+# conda env create -f environment.yml
+name: sarek-core
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+
+dependencies:
+  - bcftools=1.8
+  - bwa=0.7.17
+  - conda-forge::openjdk=8.0.144 # Needed for FastQC docker - see bioconda/bioconda-recipes#5026
+  - fastqc=0.11.7
+  - freebayes=1.2.0
+  - gatk4=4.0.3.0
+  - htslib=1.7
```

Comment:
You should use the 1.8 here.
If I remember well, `htslib`, `bcftools` and `samtools` can all have the same version

Reply:
OK, done but will check since I got a feeling that 1.8 has compatibility issues

Reply:
OK, strange, but good to know if you can confirm that

Reply:
There is even 1.9 out already! You could already skip 1.8 ... 
https://github.com/samtools/samtools/releases/